src/plugins/contextMenu/test/contextMenu.e2e.js

```diff
@@ -1,5 +1,8 @@
+/* global Handsontable */
+/* eslint no-restricted-globals: ["off", "Handsontable"] */
+
```

Comment:
Handsontable as global is already defined in our eslint rules is this is necessary?

After removing this two lines the eslint pass succesfuly.